gnocchi/storage/__init__.py

```diff
@@ -146,6 +146,14 @@ def get_driver_class(namespace, conf):
                                 conf.driver).driver
 
 
+def get_incoming_driver(conf):
+    """Return configured incoming driver only
+
+    :param conf: incoming configuring only (not global)
+    """
+    return get_driver_class('gnocchi.incoming', conf)(conf)
```

Comment:
hmm.. i was going to to make conf param already be conf.incoming but it seems i forgot. i can either pass in whole conf and use conf.incoming here or pass in just conf.incoming from cli... any preference?